content/docs/reference/machine.md

```diff
@@ -320,6 +320,34 @@ func (uart *UART) WriteByte(c byte) error
 
 Write a single byte to the UART output.
 
+## Watchdog
+
+```go
+type WatchdogConfig struct {
+	TimeoutMillis uint32
+}
+```
+
+The `WatchdogConfig` struct contains configuration for the watchdog peripheral.
+
+  * `TimeoutMillis` is the requested maximum delay between calls to `Update`.
```

Comment:
`Millis` is probably just fine. Nanoseconds are used in many places so might have been better for consistency but considering the usual range of watchdogs, milliseconds are probably just fine in practice.